deploy/README.md

```diff
@@ -0,0 +1,16 @@
+# Deployment information
```

Comment:
Thank you for this description, if you don't mind, I'd add also some instructions in the main README.